doc/release/1.10.0-notes.rst

```diff
@@ -26,61 +26,253 @@ For running on PyPy, PyPy3 6.0+ is required.
 Highlights of this release
 **************************
 
+- A new dedicated datasets submodule (`scipy.datasets`) has been added, and is
+  now preferred over usage of `scipy.misc` for dataset retrieval.
+- A new `scipy.interpolate.make_smoothing_spline` function was added. This
+  function constructs a smoothing cubic spline from noisy data, using the
+  generalized cross-validation (GCV) criterion to find the tradeoff between
+  smoothness and proximity to data points.
 
 ************
 New features
 ************
 
+`scipy.datasets` introduction
+=============================
+- A new dedicated datasets submodule moving all the dataset methods from
+  `scipy.misc` has been added.
+- The new submodule includes: `scipy.datasets.face`, `scipy.datasets.ascent`,
+  `scipy.datasets.electrocardiogram`
+- The submodule is based on ``Pooch`` (an optional dependency for SciPy), a
+  Python package to simplify fetching data files. This move will facilitate
+  SciPy to cut the sdist/wheel in size by decoupling the data files and moving
+  them out of the SciPy repository, hosting them externally and downloading
+  them when requested. After downloading the datasets once, the files are
+  cached to avoid network dependence and repeated usage.
+
+
+`scipy.integrate` improvements
+==============================
+- Added `scipy.integrate.qmc_quad`, which performs quadrature using Quasi-Monte
+  Carlo points.
+- Added parameter ``complex_func`` to `scipy.integrate.quad`, which can be set
+  ``True`` to integrate a complex integrand.
+
+
 `scipy.cluster` improvements
 ============================
 
 
 `scipy.interpolate` improvements
 ================================
+- `scipy.interpolate.interpn` now supports tensor-product interpolation methods
+  (``slinear``, ``cubic``, ``quintic`` and ``pchip``)
+- Tensor-product interpolation methods (``slinear``, ``cubic``, ``quintic`` and
+  ``pchip``) in `scipy.interpolate.interpn` and
+  `scipy.interpolate.RegularGridInterpolator` now allow values with trailing
+  dimensions.
+- `scipy.interpolate.RegularGridInterpolator` has a new fast path for
+  ``method="linear"`` with 2D data, and ``RegularGridInterpolator`` is now
+  easier to subclass
+- `scipy.interpolate.interp1d` now can take a single value for non-spline
+  methods.
+- A new ``extrapolate`` argument is available to ``BSpline.design_matrix``,
+  allowing extrapolation based on the first and last intervals.
+- A new function ``make_smoothing_spline`` has been added. It is an
```

Comment:
There are probably dozens of places where double-backticks could be changed to single backticks with fully-qualified names so that the links render. ~~LMK if you want me to go through all of these.~~ I'll go ahead and hunt them down, because I think it's helpful to have the link for those who want to learn more.
```suggestion
- A new function `scipy.interpolate.make_smoothing_spline` has been added. It is an
```